sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreCache.java

```diff
@@ -41,13 +41,18 @@
 
     private static StateStoreCache instance = null;
 
+    @VisibleForTesting
     protected final StateStore store;
 
+    @VisibleForTesting
```

Comment:
StateStoreCache already has a `fetchFrameworkId()` method we can use in the tests.  Why do the tests need to access the instance variable directly?